jerry-ext/module/module.c

```diff
@@ -180,29 +193,60 @@ jerryx_module_native_resolver (const jerry_char_t *name, /**< name of the module
  *   - an error indicating that something went wrong during the attempt to load the module.
  */
 jerry_value_t
-jerryx_module_resolve (const jerry_char_t *name, /**< name of the module to load */
-                       const jerryx_module_resolver_t *resolvers_p, /**< list of resolvers */
-                       size_t resolver_count) /**< number of resolvers in @p resolvers */
+jerryx_module_resolve (const jerry_value_t name, /**< name of the module to load */
+                       const jerryx_module_resolver_t **resolvers_p, /**< list of resolvers */
+                       const jerry_size_t count) /**< length of list */
 {
   size_t index;
-  jerry_value_t ret;
-  jerry_value_t instances = *(jerry_value_t *) jerry_get_context_data (&jerryx_module_manager);
-  jerry_value_t module_name = jerry_create_string_from_utf8 (name);
+  jerry_value_t ret = 0;
+  const jerryx_module_resolver_t *resolver_p;
+  jerry_value_t instances;
+  jerry_value_t canonical_names_static[5];
+  jerry_value_t *canonical_names = canonical_names_static;
+
+  if (!jerry_value_is_string (name))
+  {
+    ret = jerryx_module_create_error (JERRY_ERROR_COMMON, module_name_not_string, name);
+    goto bail;
+  }
+
+  if (count > 5)
+  {
+    canonical_names = jmem_heap_alloc_block_null_on_error (count * sizeof (jerry_value_t));
+    if (canonical_names == NULL)
+    {
+      goto bail;
+    }
+  }
+
+  instances = *(jerry_value_t *) jerry_get_context_data (&jerryx_module_manager);
+
+  for (index = 0; index < count; index++)
+  {
+    canonical_names[index] = 0;
+  }
 
   /* Return the cached instance if present. */
-  if (jerryx_module_check_cache (instances, module_name, &ret))
+  for (index = 0; index < count; index++)
   {
-    goto done;
+    resolver_p = resolvers_p[index];
+    canonical_names[index] =
+    resolver_p->get_canonical_name == NULL ? jerry_acquire_value (name) :
+    resolver_p->get_canonical_name (name);
```

Comment:
Interesting alignment.

Reply:
If I move the else under `jerry_acquire_value ()` I get this from check-vera.sh:
```
./jerry-ext/module/module.c:235: error: Indentation: 45 -> 4.
```
So, that's why it's like this.

Reply:
Use round brackets.